### PR TITLE
Added some more DataSeeding based tests, disabled the old ones

### PR DESCRIPTION
**Pull request by davidvarga93**
refs: MBL-15484
affects: Teacher
release note: none

test plan:

### CODE REVIEW

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 91.15% | 91.14% | 0.01%

[MBL-15484](https://instructure.atlassian.net/browse/MBL-15484)

#### Affected Apps: Teacher
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by davidvarga93** [comment]
Screenshot 2022-03-30 at 10 39 30

**Comment by inst-danger** [comment]
<details><summary>Student.ipa</summary>
  <a href="https://app.bitrise.io/app/88a3641e29e4745c/build/e60cfc5e-666e-41a3-856e-980f5d6466e9/artifact/efc6b31d18b97893/p/eccb2b5a6bedfad4bb602c5921e2d479">
    
  </a>
</details>
<details><summary>Teacher.ipa</summary>
  <a href="https://app.bitrise.io/app/88a3641e29e4745c/build/e60cfc5e-666e-41a3-856e-980f5d6466e9/artifact/48d60cfe5a96ebc5/p/673a1ac33ae47cf8d2cef0b999b006b4">
    
  </a>
</details>
<details><summary>Parent.ipa</summary>
  <a href="https://app.bitrise.io/app/88a3641e29e4745c/build/e60cfc5e-666e-41a3-856e-980f5d6466e9/artifact/31e6e0dcb24bb1de/p/28051f140073a6fac5cfe1c489f25fc6">
    
  </a>
</details>

**Review by balintbartok — DISMISSED** [review]

File: rn/Teacher/ios/TeacherE2ETests/Assignments/DSContextCardE2ETests.swift [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        let assignmentName2 = "Assignment 2"
+        let assignmentDescription2 = "Assignment 2 Description"
+        let assignment2 = seeder.createAssignment(courseId: course.id, assignementBody: .init(name: assignmentName2, description: assignmentDescription2, published: true, points_possible: 10))
+
+        let assignmentName3 = "Assignment 3"
+        let assignmentDescription3 = "Assignment 3 Description"
+        let assignment3 = seeder.createAssignment(courseId: course.id, assignementBody: .init(name: assignmentName3, description: assignmentDescription3, published: true, points_possible: 5))
+
+        // Seed submissions
+        let submission = seeder.createSubmission(courseId: course.id, assignmentId: assignment.id, requestBody:
```
Comment (vargaat) [inline rn/Teacher/ios/TeacherE2ETests/Assignments/DSContextCardE2ETests.swift:44]: These lines generate a lot of compiler warnings.

File: rn/Teacher/ios/TeacherE2ETests/Assignments/DSTodosE2ETests.swift [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        XCTAssertEqual(TabBar.todoTab.value(), todoBadgeValue)
+        XCTAssertTrue(app.find(label: oneNeedsGradingLabel).exists())
+
+        // Check if the todo counter disappears
+        seeder.postGrade(courseId: course.id, assignmentId: assignment.id, userId: student.id, requestBody: .init(posted_grade: "7"))
+        pullToRefresh()
+        pullToRefresh()
+        XCTAssertNotEqual(TabBar.todoTab.value(), todoBadgeValue)
+        XCTAssertFalse(app.find(label: oneNeedsGradingLabel).exists())
+
```
Comment (vargaat) [inline rn/Teacher/ios/TeacherE2ETests/Assignments/DSTodosE2ETests.swift:62]: Empty line could be removed.

**Review by vargaat — CHANGES_REQUESTED** [review]

**Review by vargaat — APPROVED** [review]

**Review by idevainstr — APPROVED** [review]

**Pull request merged**